src/main/java/org/assertj/core/internal/Urls.java

```diff
@@ -138,4 +146,16 @@ public void assertHasNoParameter(AssertionInfo info, URL actual, String name, St
     }
   }
 
+  public void assertIsEqualToWithSortedQueryParameters(AssertionInfo info, URL actual, URL expected) {
+    assertNotNull(info, actual);
+    if (!extractNonQueryPart(expected).equals(extractNonQueryPart(actual)))
+      throw failures.failure(info, shouldBeEqualToWithSortedQueryParameters(actual, expected));
+
+    String[] expectedQueryParams = (expected.getQuery() == null ? "" : expected.getQuery()).split("&");
+    String[] actualQueryParams = (actual.getQuery() == null ? "" : actual.getQuery()).split("&");
```

Comment:
remove unnecessary `()` and extract to a static method call `extractQueryParams`